refinery/ui/source/js/commons/services/backend-object-deletion.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
```

Comment:
Important to have unit tests for common services. Should be easy to write, given most of the common services already have unit tests.